<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchUi</class>
 <widget class="QWidget" name="SearchUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="BtnName">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>道路名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BtnLink">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LinkId</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BtnClass">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Class番号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BtnBranch">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>岔路数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LabelTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请选择查找关键字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>输入搜索内容：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnStart">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始检索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnClear">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空内容</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelResult">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>结果显示：</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>651</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>检索方式：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>259</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="BtnHash">
      <property name="text">
       <string>顺序查找</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BtnBinary">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>二分查找</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
